numpy/core/fromnumeric.py

```diff
@@ -2529,10 +2641,16 @@ def prod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     >>> np.prod(x).dtype == int
     True
 
+    You can also initialize the product by a different value.
```

Comment:
I'd argue it's "initialize with", not "initialize by"

Reply:
Done!